scapy/arch/windows/__init__.py

```diff
@@ -72,7 +74,7 @@ def _vbs_exec_code(code):
 
 def _vbs_get_iface_guid(devid):
     try:
-        # devid = str(int(devid) + 1)
+        devid = str(int(devid) + 1) # devid in the registry seems not aligned on WinXPSP3 with wmi's fields DeviceID and/or Index #TODO check if it is always the case and what happens when there are multiple interfaces on the machine
```

Comment:
I installed [npcap](https://github.com/nmap/npcap) and I don't have a "Packet scheduler interface". My devid in the registry is 2 and I don't even have the interface ID 2 in wmi output, only 1, 3, 4, 5, ...